terminus-ui/input/src/input.component.html

```diff
@@ -148,7 +148,7 @@
       [attr.autocapitalize]="autocapitalize ? 'on' : 'off'"
       [attr.autocomplete]="autocomplete"
       [attr.autofocus]="isFocused ? '' : null"
-      [attr.disabled]="shouldBeDisabled ? '' : null"
+      [disabled]="shouldBeDisabled"
```

Comment:
you sure we don't need `null` here?

Reply:
I believe so.. I went to a codepen and tried all the different versions and this one is the only one that seemed to actually disable the input for me. Our demos would disable the form field but the actual `<input>` was not being disabled. This seemed to fix that.